docker-compose.yml

```diff
@@ -42,9 +43,12 @@ services:
             POSTGRES_PASSWORD: password
             POSTGRES_USER: imageit
             POSTGRES_DB: imageit
-    gamma_backend:
-        build:
-            context: ./backend
-            dockerfile: dev.Dockerfile
+
+    redis:
+        image: bitnami/redis:5.0.8
+        restart: always
+        environment:
+            REDIS_PASSWORD: password
+        network_mode: host
```

Comment:
Any reason for running this on the host network?

Reply:
Since the backend is running on the host, right?

Reply:
no, forwarding the port is enough in this case